collections/ansible_collections/cloudkit/config_as_code/roles/aap/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Load configuration variables
+  ansible.builtin.include_vars:
+    dir: vars/
```

Comment:
Any reason to prefer this over just using `vars/main.yaml`, which is loaded automatically?

Reply:
I just wanted to be able to split the configuration in multiple files. Do you think it's too much?

Reply:
I guess I'm ambivalent. I understand wanting to split things up; it's a shame it takes an explicit task. I guess leave it for now.